api/src/test/resources/application.properties

```diff
@@ -0,0 +1,23 @@
+# H2 in-memory database configuration
+spring.datasource.url=jdbc:h2:mem:testdb
+spring.datasource.driverClassName=org.h2.Driver
+spring.datasource.username=sa
+spring.datasource.password=
+spring.datasource.platform=h2
+spring.h2.console.enabled=true
+spring.h2.console.path=/h2-console
```

Comment:
this is for testing right? if yes, move them to a test profile aka "application-test.properties" and reference them where you're using them

Reply:
It's already in src/test/, it's basic setup for any integration tests. Is test profile required here?